pkg/build/api/types.go

```diff
@@ -937,6 +941,10 @@ type BuildRequest struct {
 	// Env contains additional environment variables you want to pass into a builder container.
 	Env []kapi.EnvVar
 
+	// Args contains any build arguments that are to be passed to Docker.
```

Comment:
@smarterclayton Looking for a different solution, maybe we should have a strategy specific struct in BuildRequest that doesn't always have to be present? Spoke with @csrwng about it for a bit, seems that this may be a good option.